.github/workflows/gh-pages.yaml

```diff
@@ -4,8 +4,8 @@ on:
     branches: [ main ]
 concurrency: gh-pages
 jobs:
-  build:
-    name: "Build Element-Web"
+  pages:
+    name: "Build & Deploy Github Pages"
```

Comment:
Not related to this PR, but since I am having a look on this file, it appears that line 20 and 23 are very similar :) `uses: snok/install-poetry@v1`.

Reply:
See latest commit

Reply:
ha it's not the case anymore. Please ignore!